### PR TITLE
Add compatibility for Symfony 5

### PR DESCRIPTION
**Pull request by Deuchnord**
#hackdayParis

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "description": "Lets you find a path to the system CA bundle, and includes a fallback to the Mozilla CA bundle.",
     "type": "library",
     "license": "MIT",
+    "minimal-dependency": "dev",
```
Comment (garas) [inline composer.json]:
```suggestion
    "minimum-stability": "dev",
```

https://getcomposer.org/doc/04-schema.md#minimum-stability

File: composer.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
     "description": "Lets you find a path to the system CA bundle, and includes a fallback to the Mozilla CA bundle.",
     "type": "library",
     "license": "MIT",
+    "minimum-stability": "dev",
```
Comment (GrahamCampbell) [inline composer.json]: 👎

Comment (GrahamCampbell) [reply composer.json]: Why has this been added?

Comment (Deuchnord) [reply composer.json]: Sorry, I actually thought it was necessary for the next developments for the Sf5 compatibility, but asked to a colleague, it isn't. I originally added it to install Sf5 on my local machine and run the tests.
Fixing that :)

Comment (dunglas) [reply composer.json]: It's necessary to test locally with Symfony 5 (which hasn't been released yet, but is about to be), but this must be removed from the PR.

**Pull request merged**